lib/caesar.rb

```diff
@@ -1,6 +1,44 @@
 class Caesar
-  attr_reader :key
-
+  attr_reader :character_hash
   def initialize
+    @characters = ("a".."z")
+    @character_hash = {}
+  end
+
+  def rotate_characters(rotation)
+    @characters.to_a.rotate(rotation)
+  end
+
+  def cypher(rotation)
+    rotated = rotate_characters(- rotation)
+    @character_hash = Hash[@characters.zip(rotated)]
+    @character_hash[" "] = " "
```

Comment:
You say it's hacky, but I love what you did with the Hash! Adding the " " to the hash is a very simple way to get around that character. I can see you're logic here... we are mapping characters to their encodings, and that's what a hash does. It maps one value to another. One of the cooler ways I've seen to do this.